server/config/disaster_dashboard/Earth.textproto

```diff
@@ -394,3 +425,99 @@ categories {
     }
   }
 }
+
+categories {
+  title: "Extreme Heat"
+  blocks {
+    columns {
+      tiles {
+        type: DISASTER_EVENT_MAP
+        disaster_event_map_tile_spec: {
+          event_type_keys: "heat"
+        }
+      }
+      # tiles {
+      #   type: HISTOGRAM
+      #   title: "Number of extreme heat events last year"
+      #   stat_var_key: "heat_count"
+      # }
+      # tiles {
+      #   type: HISTOGRAM
+      #   title: "Total area affected by extreme heat events last year"
+      #   stat_var_key: "heat_area"
+      # }
+    }
+    columns {
+      tiles {
+        type: TOP_EVENT
+        title: "Most severe heat differences in the last year"
+        top_event_tile_spec {
+          event_type_key: "heat"
+        }
+      }
+    }
+  }
+
+  stat_var_spec {
+    key: "heat_count"
+    value {
+      stat_var: "Count_HeatTemperatureEvent"
+      name: "Count of extreme heat events"
+    }
+  }
+  stat_var_spec {
+    key: "heat_area"
+    value {
+      stat_var: "Area_HeatTemperatureEvent"
+      name: "Area affected by extreme heat events"
+    }
+  }
+}
+
+categories {
+  title: "Extreme Cold"
+  blocks {
+    columns {
+      tiles {
+        type: DISASTER_EVENT_MAP
+        disaster_event_map_tile_spec: {
+          event_type_keys: "cold"
+        }
+      }
+      # tiles {
+      #   type: HISTOGRAM
+      #   title: "Number of extreme cold events last year"
+      #   stat_var_key: "cold_count"
+      # }
+      # tiles {
+      #   type: HISTOGRAM
+      #   title: "Total area affected by extreme cold events last year"
+      #   stat_var_key: "cold_area"
+      # }
+    }
+    columns {
+      tiles {
+        type: TOP_EVENT
+        title: "Most severe cold differences in the last year"
```

Comment:
nit: similar suggestion for title of these cold events as made about for heat events

Reply:
done